test/utilities/apexUtils.ts

```diff
@@ -0,0 +1,169 @@
+/*
+ * Copyright (c) 2023, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import {
+  TextEditor
+} from 'wdio-vscode-service';
+import {
+  runCommandFromCommandPrompt
+} from './commandPrompt';
+import {
+  pause
+} from './miscellaneous';
+
+export async function createApexClassWithTest(name: string): Promise<void> {
+  const workbench = await browser.getWorkbench();
+  let textEditor: TextEditor;
+
+  // Using the Command palette, run SFDX: Create Apex Class to create the main class
+  const inputBox = await runCommandFromCommandPrompt(workbench, 'SFDX: Create Apex Class', 1);
+
+  // Set the name of the new Apex Class
+  await inputBox.setText(name);
+  await inputBox.confirm();
+
+  // Select the default directory (press Enter/Return).
+  await inputBox.confirm();
+  await pause(1);
+
+  // Modify class content
+  const editorView = workbench.getEditorView();
+  textEditor = await editorView.openEditor(name + '.cls') as TextEditor;
+  const classText = [
+    `public with sharing class ${name} {`,
```

Comment:
Since the text is a mixture of back tick strings and ' strings, it's easy to have a bug here.  For that reason I think it would be better to have all string here use only back ticks.